### PR TITLE
Rename DomainTotals, fields

### PR DESCRIPTION
**Pull request by vbrown608**
A little bit of renaming in advance of more work on this part of the app!

DomainTotals -> AggregatedScan
(easier to pluralize and better mirrors the "scans" table)
MTASTSTesting -> MTASTSTestingList
MTASTSEnforce -> MTASTSEnforceList
I also want to store a count, since we won't persist the full list of domains and may not capture the full list at all when it hopefully gets long. The other fields store a count, so I'm making "list" the special case that gets a special name.

### CODE REVIEW

File: db/scripts/init_tables.sql [hunk 1]
```diff
@@ -100,3 +100,5 @@ ALTER TABLE domain_totals DROP COLUMN IF EXISTS connected;
 ALTER TABLE domain_totals ADD COLUMN IF NOT EXISTS with_mxs INTEGER DEFAULT 0;
 
 ALTER TABLE domains ADD COLUMN IF NOT EXISTS mta_sts BOOLEAN DEFAULT FALSE;
+
+ALTER TABLE domain_totals RENAME TO aggregated_scans;
```
Comment (sydneyli) [inline db/scripts/init_tables.sql]: Do we need to `IF NOT EXISTS` here?

File: db/sqldb.go [hunk 2]
```diff
@@ -379,3 +379,5 @@ func (db *SQLDatabase) PutHostnameScan(hostname string, result checker.HostnameR
                                 VALUES($1, $2, $3)`, hostname, result.Status, string(data))
 	return err
 }
+
+// func (db *SQLDatabase) PutAggregatedScan
```
Comment (sydneyli) [inline db/sqldb.go]: nit: remove?

File: db/scripts/init_tables.sql (hunk 1 above)
Comment (vbrown608) [reply db/scripts/init_tables.sql]: Oh, yes! We could add a second run of the migration to our travis config to confirm that it's idempotent :D

Comment (sydneyli) [reply db/scripts/init_tables.sql]: Good idea!! :D i can do that if you haven't already

**Pull request merged**